File: MakeAWish/app/src/main/java/tt/co/jesses/makeawish/SettingsActivity.java
package tt.co.jesses.makeawish;

import com.google.firebase.analytics.FirebaseAnalytics;

import android.os.Bundle;
import android.preference.PreferenceActivity;

/**
 * Created by jessescott on 2017-02-19.
 */

public class SettingsActivity extends PreferenceActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        getFragmentManager().beginTransaction()
                .replace(android.R.id.content, new SettingsFragment())
                .commit();

        FirebaseAnalytics.getInstance(this.getApplicationContext()).setCurrentScreen(SettingsActivity.this, "SettingsActivity", SettingsActivity.class.getSimpleName());

    }

}

File: MakeAWish/app/src/main/java/tt/co/jesses/makeawish/SettingsActivity.kt
package tt.co.jesses.makeawish

import android.os.Bundle
import androidx.appcompat.app.AppCompatActivity
import com.google.firebase.analytics.FirebaseAnalytics

/**
 * Created by jessescott on 2017-02-19.
 */

class SettingsActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        supportFragmentManager.beginTransaction()
                .replace(android.R.id.content, SettingsFragment())
                .commit()

        FirebaseAnalytics.getInstance(this.applicationContext).setCurrentScreen(this@SettingsActivity, "SettingsActivity", SettingsActivity::class.java.simpleName)

    }

}
